Storage/Storage/Tools/StorageType+Extensions.swift

```diff
@@ -537,4 +537,22 @@ public extension StorageType {
     func loadPaymentGatewayAccount(siteID: Int64, gatewayID: String) -> PaymentGatewayAccount? {
         let predicate = \PaymentGatewayAccount.siteID == siteID && \PaymentGatewayAccount.gatewayID == gatewayID
         return firstObject(ofType: PaymentGatewayAccount.self, matching: predicate)
-    }}
+    }
+
+    // MARK: - System plugins
+
+    /// Returns all stored system plugins for a provided `siteID`.
+    ///
+    func loadSystemPlugins(siteID: Int64) -> [SystemPlugin] {
+        let predicate = \SystemPlugin.siteID == siteID
```

Comment:
Do you have any opinion on building keypaths like this?

Reply:
I think is a well-readable way to do it! I like it